frontend/src/modules/LoadTestContentRedux.js

```diff
@@ -1,5 +1,7 @@
 import { recursivelyCreateAddressBook } from "../helpers/transformations";
 
+let AUTH_TOKEN = localStorage.getItem("auth_token");
```

Comment:
This should be a const.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/461/commits/ff8358a9b224cffbbe037985492117466483d41d